addons/mail/static/src/chatter/web/chatter_patch.js

```diff
@@ -192,6 +198,29 @@ patch(Chatter.prototype, {
         return _t("Unfollow");
     },
 
+    changeThread(threadModel, threadId, webRecord) {
```

Comment:
it is actually not changing thread anymore

Reply:
it is only changing the name of the thread, I would suggest to rename the method